pkg/v1/cli/pluginmanager/manager.go

```diff
@@ -425,6 +483,80 @@ func SyncPlugins(serverName string) error {
 	return nil
 }
 
+// InstallPluginsFromLocalSource installs plugin from local source directory
+func InstallPluginsFromLocalSource(pluginName, version, localPath string) error {
+	// Set default local plugin distro to localpath as while installing the plugin
+	// from local source we should take t
+	common.DefaultLocalPluginDistroDir = localPath
+
+	plugins, err := DiscoverPluginsFromLocalSource(localPath)
+	if err != nil {
+		return err
+	}
+
+	found := false
+
+	errList := make([]error, 0)
+	for idx := range plugins {
+		if pluginName == plugins[idx].Name || pluginName == cli.AllPlugins {
```

Comment:
I would switch the order of this `||`. If it is AllPlugins, you may as well short circuit the rest of the if.